PayloadPkg/Library/PayloadEntryLib/PayloadEntryLib.c

```diff
@@ -78,20 +80,32 @@ PayloadInit (
   GetPayloadReservedRamRegion (&RsvdBase, &RsvdSize);
   ASSERT ((RsvdBase & EFI_PAGE_MASK) == 0);
 
+  if (FeaturePcdGet (PcdDmaProtectionEnabled)) {

```

Comment:
Need update comments in line 69 to match new changes.

Reply:
Good point,  let me update that.

Reply:
updated patch